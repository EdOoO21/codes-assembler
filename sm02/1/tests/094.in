-627 -746 687 -427 -381 -118 738 -966 407 20 -241 949 -375 -123 218 708 -439 966 100 -959 -109 -974 523 -108 -269 965 886 -70 -283 633 -563 375 480 -908 -158 -46 725 318 702 -673 -464 -730 -812 366 971 -385 -29 414 -433 771 -845 662 -233 -118 612 292 -286 -918 118 970 585 -962 -433 367 775 -538 -917 778 -79
