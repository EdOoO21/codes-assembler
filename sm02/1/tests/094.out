-79
778
-917
-538
775
367
-433
-962
585
970
118
-918
-286
292
612
-118
-233
662
-845
771
-433
414
-29
-385
971
366
-812
-730
-464
-673
702
318
725
-46
-158
-908
480
375
-563
633
-283
-70
886
965
-269
-108
523
-974
-109
-959
100
966
-439
708
218
-123
-375
949
-241
20
407
-966
738
-118
-381
-427
687
-746
-627
